packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -175,9 +139,44 @@ const DtmfTones: DtmfTone[] = [
   'Pound'
 ];
 
+const DialpadButton = (props: {
+  primaryContent: string;
+  secondaryContent?: string;
+  styles?: DialpadStyles;
+  index: number;
+  onClick: (input: string, index: number) => void;
+  onLongPress: (input: string, index: number) => void;
+}): JSX.Element => {
+  const theme = useTheme();
+  const clickFunction = () => {
+    props.onClick(props.primaryContent, props.index);
+  };
+  const longPressFunction = () => {
+    props.onLongPress(props.primaryContent, props.index);
+  };
+
+  const { handlers } = useLongPress(clickFunction, longPressFunction);
```

Comment:
both clickFunction and longPressFunction should be memoized with useCallback if being passed into a hook

Reply:
Updated